build/bootstrap.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+###
+# This script handles provisioning of the Delivery Optimization client components on supported platforms
+###
+
+# bootstrap scripts will exit immediately if a command exits with a non-zero status
+set -e
+
+# Defaults 
+INSTALL=all
+
+usage() {
+    cat <<EOM
+$(basename $0) - Script to setup development environments for Delivery Optimization
+Usage: $(basename $0) --platform <platform to install for> --install <install command>
+    --platform          # Platform to provision, supported platforms: Ubuntu1804, Ubuntu 2004, Debian9, Debian 10. Default is Ubuntu1804
```

Comment:
># Platform to provision, supported platforms: Ubuntu1804, Ubuntu 2004, Debian9, Debian 10. Default is Ubuntu1804 [](http://example.com/codeflow?start=24&length=112)

Do these comments get printed to the terminal? If not, consider changing them to normal strings that get printed.

Reply:
Yes they get printed

Reply:
Ok, then change the casing on the platform strings (all lowercase).